README.md

```diff
@@ -104,7 +104,7 @@ Please refer the /outputs.tf file for the outputs that you can get with the `ter
 
 ## Requirements
 ### Terraform plugins
-- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
+- [Terraform](https://www.terraform.io/downloads.html) 0.12.x
```

Comment:
We should add a note to the README along the lines of this one: https://github.com/terraform-google-modules/terraform-google-network/pull/55

Reply:
Got it